{% extends "colt/base.html" %}

{% load static from staticfiles %}

{% block title %}Urchin Kinome{% endblock %}
{% block meta-description%}
<meta name="description" content = "Detailed analysis of the sea urchin (Strongylocentrotus purpuratus) kinome, it's full complement of protein kinases">
{% endblock %}
{% block meta-keywords %}
<meta name="keywords" content="kinase, protein kinase, kinome, signal transduction, phosphorylate, phosphorylation, kinases, protein kinases, receptor, cytoplasmic, signaling, signalling, intracellular, model organism,  gene, sequence, protein, tyrosine kinase, serine, threonine, tyrosine, kinome, genomic, genome, database, ciliate, Strongylocentrotus, purpuratus, urchin, deuterostome, echinoderm, echinodermata">
{% endblock %}

{% block content %}

<div class="page-header">
	<h3 class="pagination-centered">The Urchin Kinome</h3>
</div>

<div class="row-fluid spacer-min"></div>

<div class="row-fluid">As part of the sequencing and analysis of
	the first echinoderm genome, we performed an analysis of the
	Strongylocentrotus purpuratus kinome.</div>

<div class="row-fluid spacer-min"></div>

<div class="row-fluid">
		<blockquote>
			<p>
				<strong>The Sea Urchin Kinome: A First Look</strong>
			</p>
			<p class="medium-font">Bradham, CA, Foltz, KR, Beane, WS, Arnone, MI, Rizzo, F, Coffman, JA, Mushegian, A, Goel, M, Morales, J, Geneviere, A, Lapraz, F, Robertson, AK, Kelkar, H, Loza-Coll, M, Townley, IK, Raisch, M, Rouz, MM, Legpage, T, Gache, C, McClay, DR, Manning, G. (2006).</p>
			<p class="medium-font">
				<strong>Dev. Biol.</strong> 300:180-93. (<a
					href="http://www.ncbi.nlm.nih.gov/pubmed/17027740">Medline</a>, <a
					href="http://linkinghub.elsevier.com/retrieve/pii/S0012160606011420">Full Text</a>)
			</p>
		</blockquote>
</div>

<div class="row-fluid spacer-min"></div>

<div class="row-fluid">
	<h3>Highlights of the Urchin Kinome</h3>
	<p>The urchin occupies a key evolutionary position which casts light on
	vertebrate evolution. It belongs to the phylum Echinodermata
	(Echinoderms), which are the closest invertebrates to the chordate
	phylum, and so a starting point to understand chordate- and
	vertebrate-specific features.</p>
	<p>
		In our previous comparison of human, insect, and nematode kinases, it
		appeared that 13 kinase subfamilies were vertebrate-specific. We now
		find homologs of 9 of these families (BCR, G11, DNAPK, HUNK, Trio,
		Lmr, Tie, RIPK, NKF3) in urchin, showing that they developed earlier
		than expected, and that there was much less invention of new classes
		of kinases within vertebrates. Also, unlike both <em>Drosophila</em> and
		<em>C. elegans</em>, which have both lost many kinase classes found in
		their common ancestor, urchins appear to have lost none, making it a
		simple non-redundant model for vertebrate kinases.</p>
</div>

<div class="row-fluid">
	<h3>Urchins have rich but non-redundant kinase set</h3>
	<p>The other major feature of the Urchin kinome is its simplicity. Despite
	having 183 of the 187 human subfamilies, it has only 353 kinases,
	compared with the 518 found in human. Most subfamilies have just one
	urchin member, but have several in vertebrates; much of this is
	probably due to the two rounds of whole-genome duplication that
	probably happened early in vertebrate evolution. Tiling array studies
	show that 88% of kinases are expressed in the embryo, which is
	experimentally well studied, and suggest that this may form a simple
	model system for the study of vertebrate-like signaling.</p>
	<p>As with other kinomes, urchin does have its own idiosyncratic
		expansions. The largest family, called just 'Urch' has 29 members and
		is remote from any other family, while other unique families are found
		in the TK and TKL groups.</p>
</div>

<div class="row-fluid">
	<h3>Urchin Kinases help classification of vertebrate kinases</h3>
	<p>
	Urchin kinases will also help subdivide complex families in human,
	based on orthology. For instance, the MAP2K and MAP3K (Ste7 and Ste11)
	families have several members in human that have not be classified into
	subfamilies. Urchin homologs now show that each family was split into 5
	subfamilies in primitive deuterostomes. Similarly, several
	'Other-Unique' singleton kinases in human are now seen to have urchin
	orthologs, showing that they have conserved and relatively ancient
	functions. These include homologs of KIS, SgK496, and SgK495. These
	data have enabled us to refine the kinase classification to provide
	family- or subfamily-level classification for all shared
	urchin-vertebrate homologs.</p>
</div>

<div class="row-fluid">
	<h3>Resources</h3>
	<p>All Urchin sequences and classification are now available through our <a
		href="{% url 'colt:kinbase' %}">KinBase</a> database, where domain structures and
	alignments can also be generated. As the genome sequence is still in
	draft form, many sequences have gaps and may include spurious
	fragments, so are not as high quality as other finished kinomes.</p>
	<p>
		More information on urchins and development is available at <a
			href="http://en.wikipedia.org/wiki/Sea_urchin">Wikipedia</a> and
		tutorials at <a href="http://www.stanford.edu/group/Urchin/">Stanford</a>
		and <a
			href="http://www.bio.davidson.edu/courses/GENOMICS/method/UrchDev.html">Davidson</a>.
		More genome information is available at the <a
			href="http://www.hgsc.bcm.tmc.edu/projects/seaurchin/">Urchin
			Genome Project</a> at Baylor, and at <a href="http://sugp.caltech.edu/">Caltech</a>.
	</p>
	
<div class="row-fluid spacer-min"></div>

	
</div>

{% endblock %}